src/components/UI/Card/Card.tsx

```diff
@@ -2,7 +2,11 @@ import React, { useContext } from 'react';
 import { ThemeContext, ThemeContextConfig, Theme } from '../../../store/theme-context';
 import styles from './Card.module.scss';
 
-const Card: React.FunctionComponent = ({ children }) => {
+interface CardProps {
+  children?: React.ReactNode;
+}
```

Comment:
no need to create extra interface, `FunctionComponent` definition already includes `children?: ReactNode | undefined`